Pouetpouets/youtube-manager
# Fix checkbox position next to channel avatar

This PR fixes the checkbox positioning in the subscription manager:

- Positions checkbox to the left of channel avatar images
- Adds proper spacing and alignment
- Improves visibility with hover effects
- Maintains clean look of the interface

Testing:
1. Visit https://www.youtube.com/feed/channels
2. Verify checkboxes appear next to channel avatars
3. Check hover effects work properly
4. Test select/unselect functionality
